source/ed/connectors/fusio_parser.cpp

```diff
@@ -1566,10 +1574,23 @@ void AdminStopAreaFusioHandler::handle_line(Data& data, const csv_row& row, bool
         throw InvalidHeaders(csv.filename);
     }
 
-    auto sa = tmp_stop_area_map.find(row[stop_area_c]);
-    if (sa == tmp_stop_area_map.end()) {
-        LOG4CPLUS_ERROR(logger, "AdminStopAreaFusioHandler : Impossible to find the stop_area " << row[stop_area_c]);
-        return;
+    std::unordered_map<std::string, ed::types::StopArea*>::iterator sa_it;
+    // For retrocompatibity
+    // TODO : to remove after the data team update, it will become useless (NAVP-1285)
+    if (!stop_id_is_present) {
+        sa_it = tmp_stop_area_map.find(row[stop_area_c]);
+        if (sa_it == tmp_stop_area_map.end()) {
+            LOG4CPLUS_ERROR(logger,
+                            "AdminStopAreaFusioHandler : Impossible to find the stop_area " << row[stop_area_c]);
+            return;
+        }
+    } else {
+        sa_it = gtfs_data.stop_area_map.find(row[stop_area_c]);
```

Comment:
Looks like the exact same code twice (if/else).
You probably don't need the `stop_id_is_present` attribute.

Reply:
if `! stop_id_is_present` we process with *tmp_stop_area_map*, not *gtfs_data.stop_area_map*